label_studio/tasks/migrations/0048_failedprediction.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.25 on 2024-08-13 18:36
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0026_auto_20231103_0020'),
+        ('ml_models', '0009_alter_thirdpartymodelversion_provider'),
+        ('ml', '0007_auto_20240314_1957'),
+        ('tasks', '0047_merge_20240318_2210'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FailedPrediction',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('message', models.TextField(blank=True, default='', help_text='The message explaining why generating this prediction failed', null=True, verbose_name='message')),
+                ('error_type', models.CharField(default=None, help_text='The type of error that caused prediction to fail', max_length=512, null=True, verbose_name='error_type')),
+                ('model_version', models.TextField(blank=True, default='', help_text='A string value that for model version that produced the failed prediction. Used in both live models and when uploading offline predictions.', null=True, verbose_name='model version')),
```

Comment:
blank=True to account for non-Prompts use cases, I take it? I figure we should do the same for model_run

Reply:
Also copying what were doing everywhere else, all TextFields have this option. We can enforce it being required in the serializer if needed, not at DB level